packages/legacy/core/App/components/tour/TourOverlay.tsx

```diff
@@ -116,7 +116,7 @@ export const TourOverlay = (props: TourOverlayProps) => {
         testID={testIdWithKey('SpotTooltip')}
         style={{ ...tooltipStyle, opacity: 1, position: 'absolute' }}
       >
-        <tourStep.render
+        <tourStep.Render
```

Comment:
Curious, what's the diff if `render` works?

Reply:
The linter complains that React components should start with a capital letter